foundry/clientSettings.d.ts

```diff
@@ -211,11 +223,6 @@ declare namespace ClientSettings {
     [key: string]: PartialMenuSetting;
   }
 
-  interface RegisteredSettings {
-    'core.combatTrackerConfig': Combat.PartialConfigSetting;
```

Comment:
I guess `Combat.PartialConfigSetting` might not be needed anymore then...?

Reply:
I didn't realize that there was a separate type for that.

Reply:
Yeah that should be removed.